pyproject.toml

```diff
@@ -136,6 +136,7 @@ markers = [
 ]
 testpaths = "tests"
 image_cache_dir = "tests/.image_cache"
+log_cli = true
```

Comment:
Where is this variable used?

Reply:
It is used by `pytest`: 

https://docs.pytest.org/en/stable/how-to/logging.html#live-logs